### PR TITLE
Support custom delete and cancel button messages

### PR DESCRIPTION
**Pull request by jace**
In the Lastuser profile page, I want to use "Remove" instead of "Delete". This patch enables that.

### CODE REVIEW

**Comment by iambibhas** [comment]
Looks fine.

File: baseframe/templates/baseframe/bootstrap3/delete.html.jinja2 [hunk 1]
```diff
@@ -8,8 +8,8 @@
   {% if form.csrf_token.errors %}
     {% for error in form.csrf_token.errors %}<div class="error">{{ error }}</div>{% endfor %}
   {% endif %}
-  <input class="btn btn-danger" type="submit" name="delete" value="{% trans %}Delete{% endtrans %}"/>
-  <input class="btn btn-default" type="submit" name="cancel" value="{% trans %}Cancel{% endtrans %}"/>
+  <input class="btn btn-danger" type="submit" name="delete" value="{% if delete_text %}{{ delete_text }}{% else %}{% trans %}Delete{% endtrans %}{% endif %}"/>
+  <input class="btn btn-default" type="submit" name="cancel" value="{% if cancel_text %}}{ cancel_text }}{% else %}{% trans %}Cancel{% endtrans %}{% endif %}"/>
```
Comment (vidya-ram) [inline baseframe/templates/baseframe/bootstrap3/delete.html.jinja2]: one bracket is missing `{ cancel_text }}`

File: baseframe/templates/baseframe/mui/delete.html.jinja2 [hunk 2]
```diff
@@ -10,8 +10,8 @@
 		  {% if form.csrf_token.errors %}
 		    {% for error in form.csrf_token.errors %}<div class="mui-form--error">{{ error }}</div>{% endfor %}
 		  {% endif %}
-		  <input class="mui-btn mui-btn--raised mui-btn--danger" type="submit" name="delete" value="{% trans %}Delete{% endtrans %}"/>
-		  <input class="mui-btn mui-btn--raised mui-btn--primary" type="submit" name="cancel" value="{% trans %}Cancel{% endtrans %}"/>
+		  <input class="mui-btn mui-btn--raised mui-btn--danger" type="submit" name="delete" value="{% if delete_text %}{{ delete_text }}{% else %}{% trans %}Delete{% endtrans %}{% endif %}"/>
+		  <input class="mui-btn mui-btn--raised mui-btn--primary" type="submit" name="cancel" value="{% if cancel_text %}}{ cancel_text }}{% else %}{% trans %}Cancel{% endtrans %}{% endif %}"/>
```
Comment (vidya-ram) [inline baseframe/templates/baseframe/mui/delete.html.jinja2]: one bracket is missing `{ cancel_text }}`

**Comment by jace** [comment]
Fixed both brackets. Anything else?

**Pull request merged**